### DISCUSSION TITLE
Running as system and User? How do we update User side.

### DISCUSSION THREAD

**Discussion by MBstatsara** [Romanitho/Winget-AutoUpdate#457] [General]
Hello

We have been using this for sometime as a system only app, but have found a lot of apps are now user side, and aren't being updated.
For some reason the scheduled task only runs as system, is this by design?

will I need to add another scheduled task to run as the user afterwards?

Also I have noticed when i run the task shortcut which  is  running the ps1 file
"C:\ProgramData\Winget-AutoUpdate\user-run.ps1""

It seems to only report in the log that its running as system.
We have noted if we are doing an install, we are using excluded_apps txt file.

This is to make sure our other apps are not patched.
Are we seeing a bug as we haven't installed from scratch in a while?

**Comment by Romanitho** [comment]
you have to reinstall with the switch `-InstallUserContext`
or use GPO to configure it
Or manually add registry key `WAU_UserContext` to `1` here:
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Winget-AutoUpdate`
Or use WiGui https://github.com/Romanitho/Winget-Install-GUI

**Reply by MBstatsara** [reply]
Thank you, will give this a go!
